torch/optim/radam.py

```diff
@@ -283,6 +292,7 @@ def _single_tensor_radam(
     eps: float,
     differentiable: bool,
     decoupled_weight_decay: bool,
+    capturable: bool,
```

Comment:
Should error if this is true, since this isn't supported, similar to ASGD, right?

Reply:
Wait...the foreach_matches_forloop eager tests pass though, so is this supported on single tensor already in eager?

Reply:
Since we don't have an error it passes.